synthtool/gcp/templates/python_library/README.rst

```diff
@@ -55,7 +55,7 @@ Code samples and snippets live in the `samples/` folder.
 
 Supported Python Versions
 ^^^^^^^^^^^^^^^^^^^^^^^^^
-Our client libraries are compatible with all current [active](https://devguide.python.org/devcycle/#in-development-main-branch) and [maintenance](https://devguide.python.org/devcycle/#maintenance-branches) versions of
+Our client libraries are compatible with all current active_ and maintenance_ versions of
 Python.
 
```

Comment:
`active_` and `maintenance_` should be in this section, not bundled with `end_of_life_` below.

Reply:
Fixed, thanks!